redis-store.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.license       = 'MIT'
 
-  s.add_dependency 'redis', '>= 2.2', '< 5'
+  s.add_dependency 'redis', '>= 3.0.6', '< 5'
```

Comment:
can you rebase from master? I updated the redis dependency to require v4 or above